proxies/live/apiproxy/targets/target.xml

```diff
@@ -18,10 +18,7 @@
         <SSLInfo>
             <Enabled>true</Enabled>
         </SSLInfo>
-        <LoadBalancer>
-            <Server name="{{ TARGET_SERVER_OVERRIDE | default('communications-manager-target') }}"/>
-        </LoadBalancer>
-        <Path>{requestpath}</Path>
+        <URL>https://comms-apim.de-adsm6.communications.national.nhs.uk</URL>
```

Comment:
Please can you revert these changes :)

Reply:
Yes I will revert them when I have approval, I just didn't want to revert them only to need to un-revert them to test any changes that anyone requested.